Gemfile.lock

```diff
@@ -310,7 +304,7 @@ GEM
     rubocop-rspec (1.37.1)
       rubocop (>= 0.68.1)
     ruby-progressbar (1.10.1)
-    rubyzip (2.0.0)
+    rubyzip (1.3.0)
```

Comment:
Is that right?

Reply:
Good spot, I did not realise that webdrivers was already on 4.x. I've updated the Gemfile.